### ISSUE TITLE
Monitor does not turn back on if 'BMBT Off' is ON

### ISSUE THREAD

**Issue by Goretex432** [tedsalmon/BlueBus#148]
I was playing around with the settings (in hardware v2, latest firmware), and wasn't sure what 'BMBT Off" was, so I turned it on. Looking in the manual it says: 
BMBT Off: Select weather or not to have the Monitor shut off automatically after the vehicle reaches 5km/h.

The monitor did turn off when I moved upto speed, but then it did not turn back on. Didn't matter what button I pressed, turn the radio off/on, the monitor stayed dark. Only a restart of the car turned the monitor back on. 

This is in a 2003 E46 M3 in the UK, with Navi + tv + telephone adapter.

**Comment by sven358** [comment]
Hello,

Since i was in my e46 320i 2003. I tried and have the same issue!

**Comment by tedsalmon** [comment]
Yeah, I assumed the GT would listen for the monitor off message, but I was wrong and that’s tracked internally by the GT. I will have to listen to the BMBT inputs and turn the monitor back on when an input is seen.
